guide/payments/payment-request-formats.md

```diff
@@ -0,0 +1,147 @@
+parent: Sending bitcoin
+has_children: false
+permalink: /guide/payments/send/payment-request-formats/
+main_classes: -no-top-padding
+image: /assets/images/guide/payments/send/bitcoin-payments-social.jpg
+
+image_base: /assets/images/guide/payments/send/
+images:
+    - file: home
+      alt:
+      caption:
+---
+
+<!--
+
+Editor's notes
+
+-->
+
+# Payment request formats
+
+Payment information can be shared in many formats and over diverse communication channels. Each has its own advantages and limitations, these formats seen together allow for broad flexibility in initiating payments. Note that some of them are still maturing and have varying support across applications.
+
+### On-chain addresses
+
+Addresses are used for transactions on the base layer. More details on the [address page]({{ '/guide/glossary/address/' | relative_url }}) in the Glossary.
+
+### Lightning invoices ([BOLT 11](https://github.com/lightning/bolts/blob/master/11-payment-encoding.md))
+
+Lightning invoices are the basic payment mechanism on the Lightning network. They are usually set to expire after 1 hour and should only be paid once for best security and privacy. Invoices have to be created by the recipient and shared with the sender who then makes the payment.
+
+### Offers ([BOLT 12](https://bolt12.org))
+
+This draft specification has similarities with to LNURL-Pay and LNURL-Withdraw, but uses the Lightning network itself as the communication channel.
+
+### [Lightning AMP invoices](https://docs.lightning.engineering/lightning-network-tools/lnd/amp) (Atomic Multi-Path)
+
+This type of invoice allow for payments to be broken up in small payments, potentially increasing the likelihood of success for larger amounts. Invoices typically expire after one day, but can also be configured to be static and be paid multiple times. Unlike BOLT 11 invoices, AMP invoices can also be initiated by the sender without action by the recipient.
+
+### Lightning node IDs
+
+Senders can initiate payments to recipients only knowing their node ID, using [Keysend](https://lightning.readthedocs.io/lightning-keysend.7.html) and AMP invoices.
+
+### [LNURL](https://coincharge.io/en/lnurl-for-lightning-wallets/)
+
+This internet-based protocol establishes techniques on top of Lightning invoices for several important use cases:
+
+**LNURL-Pay**
+
+Makes it possible to generate Lightning invoices on-demand, based on a static identifier (in contrast with regular Lightning invoices, which expire and should only be paid once).
+
+**LNURL-Withdraw**
+
+Allows for invoices to offer withdrawals with optional minimum and maximum amounts.
+
+**LNURL-Auth**
+
+Lets users authenticate to third-party services via their wallet (Lightning node).
+
+**LNURL-Channel**
+
+Streamlines the channel opening user flow.
+
+### [Lightning addresses](https://lightningaddress.com)
+
+This identifier builds on top of LNURL-Pay and introduces the familiar format of email addresses as identifiers.
+
+## Examples
```

Comment:
Could these examples not be shown with the associated entry above rather than pushed to the bottom?

Reply:
Seems reasonable

Reply:
That's how I had it at first, but the page was really choppy to follow. I found that having the examples right next to each other made them lots easier to compare, and allows the explanations to focus more on purpose, benefits, and limitations.